Yoast/wordpress-seo
# Update licenses page

## Summary

This PR can be summarized in the following changelog entry:
* Changed the licenses page to make it work with the new my.yoast.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:
* You have to add some code [here](https://github.com/Yoast/wordpress-seo/pull/7191/files#diff-6f693a5dffe9ee38c07b74afbf9ddc4bR83), or you can hack around `http_response` by setting the response value:
```
// For testing with the new layout:
$response['serverVersion'] = '2';

// For the current situation just do:
$response['serverVersion'] = '1';
```

* This test can be done for a plugin like news seo or video seo.
* Go to the license page and see the plugin button is shown as: buy ... 
* Enable the plugin.
* The plugin has will be displayed as `installed` | `not activated`
* Active the plugin for the acceptance testing environment on my.yoast.
* On the license page the `not activated` will be displayed as `activated`
* Disable the plugin for the site and refresh the license page: it should be shown as `not activated`

I've also made some changes on the license manager: https://github.com/Yoast/License-Manager/pull/109

Fixes #7177
Fixes #6561
